src/pipecat/services/simli.py

```diff
@@ -0,0 +1,116 @@
+import asyncio
+
+from pipecat.frames.frames import (
+    Frame,
+    OutputImageRawFrame,
+    TTSAudioRawFrame,
+    StartInterruptionFrame,
+    EndFrame,
+    CancelFrame,
+)
+from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
+
+import numpy as np
+from av import AudioFrame
+from av.audio.resampler import AudioResampler
+from simli import SimliClient, SimliConfig
+
+
+class SimliVideoService(FrameProcessor):
+    def __init__(
+        self, simliConfig: SimliConfig, useTurnServer=False, latencyInterval=60
+    ):
+        super().__init__()
+        self.simliClient = SimliClient(simliConfig, useTurnServer, latencyInterval)
+
+        self.pipecatResampler: AudioResampler = None
+        self.name = "SimliAi"
+        self.ready = False
+        self.simliResampler = AudioResampler("s16", 1, 16000)
+        self.AudioTask: asyncio.Task = None
+        self.VideoTask: asyncio.Task = None
+
+    async def startConnection(self):
+        await self.simliClient.Initialize()
+        self.ready = True
+        # Create task to consume and process audio and video
+        self.AudioTask = asyncio.create_task(self.consume_and_process_audio())
+        self.VideoTask = asyncio.create_task(self.consume_and_process_video())
```

Comment:
I think this could be:

```
async def _start_connection(self):
   ....
```

and then, in `process_frame` we can do:

```
if isinstance(frame, StartFrame):
    await self._start_connection()
```